<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertDNADialog</class>
 <widget class="QDialog" name="InsertDNADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Nucleic Acids</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>DNA/RNA Builder</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="typeComboBox">
        <item>
         <property name="text">
          <string>DNA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RNA</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nucleic Acids:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolButton_A">
          <property name="toolTip">
           <string>Adenine</string>
          </property>
          <property name="text">
           <string extracomment="adenine (A)">A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_C">
          <property name="toolTip">
           <string>Cytosine</string>
          </property>
          <property name="text">
           <string extracomment="cytosine (C)">C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_G">
          <property name="toolTip">
           <string>Guanine</string>
          </property>
          <property name="text">
           <string extracomment="guanine (G)">G</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_TU">
          <property name="toolTip">
           <string>Thymine</string>
          </property>
          <property name="text">
           <string extracomment="thymine (T)">T</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sequence:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="sequenceText">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QComboBox" name="bpCombo">
          <property name="toolTip">
           <string>the number of base pairs per helix turn</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Z</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="bpTurnsSpin">
          <property name="toolTip">
           <string>the number of base pairs per helix turn</string>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>15.000000000000000</double>
          </property>
          <property name="value">
           <double>10.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Bases Per Turn:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Strands:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="singleStrandRadio">
          <property name="text">
           <string extracomment="single-strand DNA">Single</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="doubleStrandRadio">
          <property name="text">
           <string extracomment="double-strand DNA">Double</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="insertButton">
       <property name="text">
        <string>Insert</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
